csharp/ql/src/semmle/code/csharp/ir/implementation/raw/Instruction.qll

```diff
@@ -743,7 +743,7 @@ class ReturnValueInstruction extends ReturnInstruction {
   final Instruction getReturnValue() { result = getReturnValueOperand().getDef() }
 }
 
-class ReturnIndirectionInstruction extends Instruction {
+class ReturnIndirectionInstruction extends VariableInstruction {
```

Comment:
Until now I thought that `ReturnIndirectionInstruction` was for simulating a read of a returned pointer. I can see now it's for simulating a read of a pointer parameter. Please add QLDoc.